docs/example_figure.py

```diff
@@ -0,0 +1,86 @@
+from matplotlib import pyplot as plt
```

Comment:
Should we put this file under "examples"?

Reply:
I put both this python file and the rst file in a dir called "examples".

In general, if you have good ideas about file/variable/function names and/or file structure, please implement them or let me know. I tend to use placeholder names to avoid getting blocked at thinking of the perfect name for a file.